rails/locales/ja.yml

```diff
@@ -0,0 +1,5 @@
+en:
```

Comment:
@grosser there is an error in this PR so it overrides default english translations with japanese texts

Reply:
I created PR #419

Reply:
5.12.1

Reply:
Sorry; Thank you.